tools/blue_deployment/prompts.js

```diff
@@ -0,0 +1,106 @@
+const Web3 = require('web3');
+const inquirer = require('inquirer');
+
```

Comment:
A short documentation at the top of the file would make it easier to understand what this is inquiring and why.